src/demo/components/styles.tsx

```diff
@@ -25,16 +24,287 @@ export const PageIntroStyle = styled.div`
   & .footer {
     margin-bottom: 2rem;
   },
-}`;
+`;
 
 export const BoxContainer = styled.div`
   text-align: center;
   box-shadow: 0 4px 10px 0 rgba(0, 0, 0, 0.1);
   border-radius: 10px;
   margin: 2rem 4rem 0rem;
-  border: 1px solid #E4E4E4;
+  border: 1px solid #e4e4e4;
   p {
     text-align: justify;
   }
+`;
+
+export const HeaderContainer = styled.div`
```

Comment:
styles.tsx designed for the common style which is used in many pages otherwise create component based css

Reply:
I created some common styles which we will use in the other pages. If you want, I can separate it.

Reply:
I changed some of the styles.